Formatters/GitHubTeams.Format.ps1xml

```diff
@@ -42,33 +59,43 @@
           <ListEntry>
             <ListItems>
               <ListItem>
+                <Label>Name</Label>
                 <PropertyName>name</PropertyName>
               </ListItem>
               <ListItem>
+                <Label>Description</Label>
                 <PropertyName>description</PropertyName>
               </ListItem>
               <ListItem>
+                <Label>Privacy</Label>
                 <PropertyName>privacy</PropertyName>
               </ListItem>
               <ListItem>
+                <Label>Permission</Label>
                 <PropertyName>permission</PropertyName>
               </ListItem>
               <ListItem>
+                <Label>Created</Label>
```

Comment:
I always find it annoying when the default view of an object shows label names that don't exist as properties on the object. I.e for this, if I try and read `$Team.Created` I will not get the desired result. I will then have to run `$Team | Fl -Property *` to get the actual property name `created_at`. Same for the updated labels below. If you want to do this, you should build the output object with these new property names.

Reply:
This is interesting feedback that I hadn't considered, and I appreciate it.  I'll make that change (and change `Parent` to be `parent.name`).